eventmesh-examples/src/main/java/org/apache/eventmesh/tcp/common/EventMeshTestUtils.java

```diff
@@ -38,6 +38,9 @@
 import java.util.UUID;
 import java.util.concurrent.ThreadLocalRandom;
 
+import javax.lang.model.element.PackageElement;
```

Comment:
Just extracting the duplicate code as a method, why would you use these two imports?

Reply:
well I think I am using vscode. And it has auto import feature. I think thats the reason.